packages/pwa/app/components/_app-config/index.jsx

```diff
@@ -88,8 +77,16 @@ const AppConfig = ({children, locals = {}}) => {
 
 AppConfig.restore = (locals = {}) => {
     // Parse the locale from the page url.
-    const locale = getLocale(locals) || DEFAULT_LOCALE
-    const currency = getPreferredCurrency(locale) || DEFAULT_CURRENCY
+    const originalUrl = locals.originalUrl
+    const url = pathToUrl(originalUrl)
+    const site = resolveSiteFromUrl(url)
+
+    apiConfig.parameters.siteId = site?.id
+    const l10nConfig = getL10nConfig(originalUrl)
```

Comment:
If we already have the site configuration object, could we simple not grab the `l10n` object from it. Do we need `getL10nConfig` anymore?